File: OLSource1/v1/VS_officedev/codesnippet/CSharp/walkthrough--complex-data-binding-in-vsto-add-in-project_5.cs
            this.addressBindingSource.DataSource = this.adventureWorksDataSet.Address;
            this.addressListObject.AutoSetDataBoundColumnHeaders = true;
            this.addressListObject.SetDataBinding(
                this.addressBindingSource, "", "AddressID", "AddressLine1", 
                "AddressLine2", "City", "StateProvince", "CountryRegion", "PostalCode");

File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/walkthrough--complex-data-binding-in-vsto-add-in-project_5.vb
        Me.addressBindingSource.DataSource = Me.adventureWorksDataSet.Address
        Me.addressListObject.AutoSetDataBoundColumnHeaders = True
        Me.addressListObject.SetDataBinding( _
            Me.addressBindingSource, "", "AddressID", "AddressLine1", _
            "AddressLine2", "City", "StateProvince", "CountryRegion", "PostalCode")